scripts/testnet/include/validators.sh

```diff
@@ -97,6 +101,21 @@ iterateOverValidatorsMultiKey() {
    (( VALIDATOR_INDEX++ ))
 }
 
+iterateOverSovereignValidatorsMultiKey() {
+  local callback=$1
+  local VALIDATOR_INDEX=$META_VALIDATORCOUNT
+
+  # Iterate over shards and start validators
+  (( max_shard_id=$SHARDCOUNT - 1 ))
+  for SHARD in $(seq 0 1 $max_shard_id); do
```

Comment:
I think we can skip pe shardCount and for execution here, since we only have one shard?

Reply:
fixed